conf/modules.config

```diff
@@ -584,18 +584,52 @@ process {
 
 process {
     withName: '.*PREPARE_MT_ALIGNMENT:SAMTOOLS_VIEW_MT' {
+        ext.args = { "-h " }
```

Comment:
I think we should try to be consistent in when to use single quote or double quote. Double quote is needed when the string contains variables that are to be interpreted, but for plain text you could do with single. I see that we have mixed and matched a little bit before but we should try to settle for a standard. Do you know if there is a standard int he community? You can leave it as is for now if you want. Just something to think about :)